tests/unit/cluster.tcl

```diff
@@ -109,13 +110,11 @@ start_server [list overrides $base_conf] {
         assert_equal {key9184688 v2} [$node1_rd read]
     }
 
-    $node1_rd close
```

Comment:
why is this close removed?

Reply:
It is moved so that these tests can be skipped using e.g. `./runtest --single unit/cluster --only 'Migrate the last slot away from a node'`. Without these changes, skipping a test case results in errors like undefined variables. (I put these changes in a separate commit.)

Reply:
but doesn't that close correspond to line 118?

Reply:
It was created as second time on line 118 but I removed that. Now it's only created once (line 93) and closed once (line 149).

Reply:
Ah sorry my bad. missed that.